setup.py

```diff
@@ -9,6 +9,7 @@
     include_package_data=True,
     install_requires=[
         'databricks-api==0.9.0',
+        'databricks-sdk',
```

Comment:
Same as above, restrict to a specific version we know works so that we avoid same pain later

Reply:
Yes, done